service_identity/src/main/kotlin/org/thechance/service_identity/api/endpoints/permissionRoutes.kt

```diff
@@ -0,0 +1,61 @@
+package org.thechance.service_identity.api.endpoints
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import org.koin.ktor.ext.inject
+import org.thechance.service_identity.api.model.PermissionDto
+import org.thechance.service_identity.domain.usecases.permission.PermissionUseCasesContainer
+
+fun Route.permissionRoutes() {
+    val permissionUseCasesContainer: PermissionUseCasesContainer by inject()
+    route("/permissions") {
+        post {
+            val permission = call.receive<PermissionDto>()
+            val success = permissionUseCasesContainer.addPermission(permission.toPermission())
+            if (success) {
+                call.respond(HttpStatusCode.Created)
+            } else {
+                call.respond(HttpStatusCode.InternalServerError)
+            }
+        }
+
+        delete("/permissions/{permissionId}") {
```

Comment:
It looks like you've duplicated the word "permissions" unnecessarily. Instead, it should be just ```delete("/{permissionId}")``` without the repetition.